app/helpers/comments_helper.rb

```diff
@@ -299,9 +301,11 @@ def commenter_id_for_css_classes(comment)
   def css_classes_for_comment(comment)
     return if comment.nil?
 
+    official = "official" if comment&.pseud&.user&.official
     unreviewed = "unreviewed" if comment.unreviewed?
     commenter = commenter_id_for_css_classes(comment)
-    "#{unreviewed} comment group #{commenter}".strip
+
+    "#{official} #{unreviewed} comment group #{commenter}".strip
```

Comment:
Would it make sense to use `squish` here instead?

Reply:
That seems like a good idea. It'll get rid of extra whitespace when a comment is official but not unreviewed.

Reply:
Just this one left -- I think it probably got overlooked because it was collapsed.

Reply:
huh, I remember changing it to squash - must have been overwritten when I merged commits